outputs.tf

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Checking the length forces the outputs to resolve after the null_resource has executed,
```

Comment:
I haven't used the null data source resource before, so I'm working on trying to get this to work. My initial attempts haven't worked, but let me wrestle with it some more and if I don't have it figured out by Monday, I'd love some input.

Reply:
I've been able to get this to work, except for an error I'm seeing below after getting a successful destroy in my tests, so if you have any thoughts on that, it'd be helpful.

Reply:
We might want to add a separate destroy hook to do a no-op in such cases?

Reply:
I did some research, and it looks like this is an open bug in Terraform: https://github.com/hashicorp/terraform/issues/17337#issuecomment-366812724

The only option I can see here is to revert back to [using the logical local variables rather than the null_data_source](https://github.com/terraform-google-modules/terraform-google-gcloud/pull/2/commits/a7ddc6cd13133a88db34c65c23ffee43c7d35ed4) that @aaron-lane suggested.

Reply:
I went ahead and did this, but also opened an issue for this repo to re-evaluate when that bug has been addressed upstream: https://github.com/terraform-google-modules/terraform-google-gcloud/issues/3

Reply:
I went ahead and did this, but also opened an issue for this repo to re-evaluate when that bug has been addressed upstream: https://github.com/terraform-google-modules/terraform-google-gcloud/issues/3